bot/exts/evergreen/realpython.py

```diff
@@ -0,0 +1,75 @@
+# Credit to Vthechamp22, who heavily inspired this bit of code
```

Comment:
To be very honest, I don't think this is necessary. If you do want to give real credit, add them as a co-author to the commits.

Reply:
@Vthechamp22 , how does that sound to you?